flexbox/src/androidTest/res/layout/recyclerview_reverse.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright 2016 Google Inc. All rights reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<android.support.v7.widget.RecyclerView android:id="@+id/recyclerview"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="400dp"
+    android:layout_height="match_parent"
+    android:orientation="horizontal"
+    app:reverseLayout="true"
+    app:stackFromEnd="true"
+    app:layoutManager="com.google.android.flexbox.FlexboxLayoutManager">
```

Comment:
RecyclerView resolves the layout manager with reflection, and this can cause problems with ProGuard. Consider adding [a consumer ProGuard settings](https://robotsandpencils.com/blog/use-proguard-android-library/).